.gitignore

```diff
@@ -0,0 +1,3 @@
+
+*.json
+poker-pals/package-lock.json
```

Comment:
You don't need to add this. Master already has code ignoring `package-lock.json`. Also, we don't wanna ignore all `json` files since `package.json` contains all of our dependencies.